test/factories/pieces.rb

```diff
@@ -39,6 +39,6 @@
     type 'Pawn'
     color 'black'
     x_coord 1
-    y_coord 1
+    y_coord 6 || 5
```

Comment:
I don't think this is doing what you expect.  The `|| 5` doesn't do anything here.  See this in IRB:

```
irb(main):004:0> var = 5 || 6
=> 5
irb(main):005:0> var
=> 5
```